spec/features/admin_interface_spec.rb

```diff
@@ -3,71 +3,86 @@
 require 'rails_helper'
 
 describe 'GET /admin', type: :feature, js: true do
-  before do
-    @questionnaire_names = ['my_name',
-                            'second_name',
-                            'hey I have spaces dagboek 5x per week dinsdag, woensdag, vrijdag']
-    @questionnaire_names.each do |questionnaire_name|
-      FactoryGirl.create(:questionnaire, name: questionnaire_name)
+  context 'exporter' do
+    before do
+      @questionnaire_names = ['my_name',
+                              'second_name',
+                              'hey I have spaces dagboek 5x per week dinsdag, woensdag, vrijdag']
+      @questionnaire_names.each do |questionnaire_name|
+        FactoryGirl.create(:questionnaire, name: questionnaire_name)
+      end
     end
-  end
-
-  it 'should give a 404 error when not authorized' do
-    visit '/admin'
-    expect(page).to have_content('HTTP Basic: Access denied.')
-  end
 
-  it 'should be able to download questionnaires correctly' do
-    basic_auth 'admin', 'admin', '/admin'
-    visit '/admin'
-
-    # People
-    expect(page).to have_content('People')
-    expect(page).not_to have_css('a[disabled]')
-    expect(page).to have_link('Download', href: '/admin/person_export.csv')
-    page.all('a', text: 'Download')[0].click
-    # expect(page.response_headers['Content-Type']).to eq 'text/csv'
-    expected_filename = "people_#{Time.zone.now.to_date}.csv"
-    # expect(page.response_headers['Content-Disposition']).to match(/attachment; filename="#{expected_filename}"/)
-    expect(page).to have_css('a[disabled]', count: 1)
-
-    # ProtocolSubscriptions
-    expect(page).to have_content('ProtocolSubscriptions')
-    expect(page).to have_link('Download', href: '/admin/protocol_subscription_export.csv')
-    page.all('a', text: 'Download')[1].click
-    # expect(page.response_headers['Content-Type']).to eq 'text/csv'
-    expected_filename = "protocol_subscriptions_#{Time.zone.now.to_date}.csv"
-    # expect(page.response_headers['Content-Disposition']).to match(/attachment; filename="#{expected_filename}"/)
-    expect(page).to have_css('a[disabled]', count: 2)
+    it 'should give a 404 error when not authorized' do
```

Comment:
`it 'should have content HTTP Basic: Access denied. when not authorized`?

Reply:
Fixed.